packages/react/src/components/OrderedList/OrderedList-story.js

```diff
@@ -1,92 +0,0 @@
-import React from 'react';
-import OrderedList from '../OrderedList';
-import { withKnobs, boolean } from '@storybook/addon-knobs';
-import ListItem from '../ListItem';
-
-const props = {
-  regular: () => {
-    return {
-      isExpressive: boolean('Expressive', false),
-    };
-  },
-};
-
-export default {
-  title: 'Components/OrderedList',
-  component: OrderedList,
-  decorators: [withKnobs],
-  subcomponents: {
-    ListItem,
-  },
-};
-
-export const Default = () => (
-  <OrderedList>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-    <ListItem>Ordered List level 1</ListItem>
-  </OrderedList>
-);
-
-export const Nested = () => {
```

Comment:
What's the reason for removing the Nested and NativeListStyles stories?

Also percy is failing - for any story name changes the `e2e/components/OrderedList/OrderedList-test.e2e.js` file will need updated.

Reply:
Ah I can add them back I though it was redundant due to the controls but no problem.

Reply:
Cool, yeah I think it's helpful from a VRT perspective to have those in there to catch any regressions